csharp/sample/Microsoft.ML.OnnxRuntime.ResNet50v2Sample/Program.cs

```diff
@@ -19,10 +17,9 @@ public static void Main(string[] args)
             string imageFilePath = args[1];
 
             // Read image
-            using Image<Rgb24> image = Image.Load<Rgb24>(imageFilePath, out IImageFormat format);
+            using Image<Rgb24> image = Image.Load<Rgb24>(imageFilePath);
```

Comment:
note: It would be possible to use here the `RgbaVector` pixel format instead of `Rgb24`, which would make sure the rgb values are normalized between 0 and 1, so the division by 255 later on in the Preprocess step would not be required.